project/src/main/kotlin/org/ionproject/core/klass/sql/KlassData.kt

```diff
@@ -41,4 +41,8 @@ internal object KlassData {
         join $SCHEMA.$COURSE as CR on C.$CID=CR.$COURSE_ID
         where C.$CID=:$CID order by C.$CAL_TERM offset :$OFFSET limit :$LIMIT
     """
+
+    const val CHECK_IF_COURSE_EXISTS = """
+        select count(*) from $SCHEMA.$COURSE as C where C.$COURSE_ID=:$CID
```

Comment:
I suggest adding a `limit 1` to stop the counting after the first row.

Reply:
But, can there be more than one course with the same courseId?

Reply:
You're right. I commented out of reflex after seeing `count(*)`. 😛